src/main/java/signatures/Utilities.java

```diff
@@ -0,0 +1,72 @@
+package signatures;
+
+import org.bouncycastle.crypto.digests.SHA256Digest;
+import org.bouncycastle.crypto.engines.RSAEngine;
+import org.bouncycastle.crypto.params.RSAKeyParameters;
+import org.bouncycastle.crypto.signers.PSSSigner;
+import org.bouncycastle.openssl.PEMKeyPair;
+import org.bouncycastle.openssl.PEMParser;
+
+import java.io.StringReader;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.security.KeyFactory;
+import java.security.KeyPair;
+import java.security.PrivateKey;
+import java.security.PublicKey;
+import java.security.spec.PKCS8EncodedKeySpec;
+import java.security.spec.RSAPrivateKeySpec;
+import java.security.spec.X509EncodedKeySpec;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class Utilities {
+
+    // Private constructor to prevent instantiation.
+    private Utilities() {
+        throw new UnsupportedOperationException();
+    }
+
+    public static String getFileBodyAt(String envVar) {
+        try {
+            String pathToFile = System.getenv(envVar);
+            if (pathToFile == null) {
+                // If the env var is unset, it presents as null.
+                throw new RuntimeException(envVar + " environment variable must be set");
+            }
+            Stream<String> streamer = Files.lines(Paths.get(pathToFile));
+            String body = streamer.collect(Collectors.joining("\n"));
```

Comment:
Is this portable?  Does this code need to run on Windows?  Why not just read the file as bytes instead of introducing this potential pitfall by reading as lines if we're going to return the entire file as a string anyway?

Reply:
I honestly don't know the answers to those questions, I don't Java enough. I believe it's a feature that's dependent on your Java version more than your OS environment? I am certainly open to suggestions but I don't Java often enough to be able to infer the code you might be thinking of. Would you be willing to post a better method if you know it?

Reply:
That looks like a fairly normal Java way of doing things.  Yes it will be portable

Reply:
Sorry, not sure why I didn't see the followup to this.  My portability concern is that if running on windows the line separator is "\r\n".  So your Files.lines() will be aware of this and break the file on whatever the local line separator is, and then you're replacing that with the UNIX newline.  You could use https://docs.oracle.com/javase/8/docs/api/java/lang/System.html#lineSeparator-- , or ideally just avoid the whole issue using something like https://docs.oracle.com/javase/8/docs/api/java/nio/file/Files.html#readAllBytes-java.nio.file.Path-.